src/cr/cube/cube.py

```diff
@@ -455,6 +455,11 @@ def _cube_response(self):
                 "(str) or dict." % type(self._cube_response_arg).__name__
             )
 
+    @lazyproperty
+    def _has_missing(self):
+        """True if cube dimensions doesn't contain MR and NUM_ARRAY else False."""
+        return not all(d in self.dimension_types for d in [DT.MR, DT.NUM_ARRAY])
```

Comment:
I have a problem interpreting the structure of `not all(. . .)`. I think it'd be easier to interpret as:
```
return DT.MR not in self.dimension_types and DT.NUM_ARRAY not in self.dimension_types
```
but that might be obsolete 🤷

Reply:
sorry but even this one is not the same of what i wrote, I'll try to simplify it in a further commit